src/shogun/regression/KRRNystrom.cpp

```diff
@@ -79,6 +79,14 @@ bool CKRRNystrom::solve_krr_system()
 {
 	int32_t n=kernel->get_num_vec_lhs();
 
+	if (n < m_num_rkhs_basis)
+	{
+		SG_SWARNING("Number of sampled rows (%d) must be less than or equal to "
```

Comment:
This is to make it work with generated tests since I can't call `set_num_rkhs_basis()`, otherwise I could set by default `m_num_rkhs_basis = 1` instead of `100`